web/cobrands/buckinghamshire/assets.js

```diff
@@ -0,0 +1,193 @@
+    http_options: {
+        params: {
+            TYPENAME: "Grit_Bins"
+        }
+    },
+    attributes: {
+        central_asset_id: 'central_as',
+        site_code: 'site_code' // different capitalisation, sigh
+    },
+    asset_category: ["Salt bin damaged", "Salt bin refill", "Grit bin issue"],
+    asset_item: 'grit bin',
+}));
+
+fixmystreet.assets.add($.extend(true, {}, defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "StreetLights_Merged"
+        }
+    },
+    attributes: {
+        central_asset_id: 'central_as',
+        site_code: 'Site_code'
+    },
+    asset_category: [
+        'Light on during the day',
+        'Street light dim',
+        'Street light intermittent',
+        'Street light not working' ],
+    asset_item: 'street light'
+}));
+
+
+// The "whole street asset" layer indicates who is responsible for maintaining
+// a road via the 'feature_ty' attribute on features.
+// These are roads that Bucks maintain.
+var bucks_types = [
+    "2", // HW: STRATEGIC ROUTE
+    "3A", // HW: MAIN DISTRIBUTOR
+    "3B", // HW: SECONDARY DISTRIBUTOR
+    "4A", // HW: LINK ROAD
+    "4B", // HW: LOCAL ACCESS ROAD
+];
+// And these are roads they don't maintain.
+var non_bucks_types = [
+    "HE", // HW: HIGHWAYS ENGLAND
+    "HWOA", // OTHER AUTHORITY
+    "HWSA", // HW: Whole Street Asset
+    "P", // HW: PRIVATE
+];
+
+// We show roads that Bucks are and aren't responsible for, and display a
+// message to the user if they click something Bucks don't maintain.
+var types_to_show = bucks_types.concat(non_bucks_types);
+
+// Some road types we don't want to display at all.
+var types_to_hide = [
+    "11", // HW: BYWAY OPEN TO TRAFFIC
+    "12", // HW: FOOTPATH PROW
+    "13", // HW: BYWAY RESTRICTED
+    "14", // HW: BRIDLEWAY
+    "9", // HW: NO CARRIAGEWAY
+];
+
+var highways_style = new OpenLayers.Style({
+    fill: false,
+    strokeColor: "#5555FF",
+    strokeOpacity: 0.1,
+    strokeWidth: 7
+});
+
+function bucks_owns_feature(f) {
+    return f &&
+           f.attributes &&
+           f.attributes.feature_ty &&
+           bucks_types.indexOf(f.attributes.feature_ty) > -1;
+}
+
+function bucks_does_not_own_feature(f) {
+    return !bucks_owns_feature(f);
+}
+
+var rule_owned = new OpenLayers.Rule({
+    filter: new OpenLayers.Filter.FeatureId({
+        type: OpenLayers.Filter.Function,
+        evaluate: bucks_owns_feature
+    })
+});
+
+var rule_not_owned = new OpenLayers.Rule({
+    filter: new OpenLayers.Filter.FeatureId({
+        type: OpenLayers.Filter.Function,
+        evaluate: bucks_does_not_own_feature
+    }),
+    symbolizer: {
+        strokeColor: "#555555"
+    }
+});
+highways_style.addRules([rule_owned, rule_not_owned]);
+
+function show_responsibility_error(id) {
+    hide_responsibility_errors();
+    $("#js-bucks-responsibility").removeClass("hidden");
+    $("#js-bucks-responsibility .js-responsibility-message").addClass("hidden");
```

Comment:
These elements aren't present in the fms.com template, so no message is being shown.